hedera-node/hedera-mono-service/src/main/java/com/hedera/services/txns/network/UpgradeActions.java

```diff
@@ -151,7 +151,12 @@ private CompletableFuture<Void> extractNow(
         return runAsync(
                 () -> {
                     try {
-                        FileUtils.cleanDirectory(new File(artifactsLoc));
+                        var artifactsLocFile = new File(artifactsLoc);
+                        if (artifactsLocFile.exists()) {
+                            FileUtils.cleanDirectory(new File(artifactsLoc));
+                        } else {
+                            artifactsLocFile.mkdirs();
+                        }
```

Comment:
@tinker-michaelj and @Neeharika-Sompalli I changed this because as it was written it would pass when all `UpgradeActionsTest` tests were run, but running a single test would fail 🤔 If you could confirm that this is correct and ok then that'd be great!

Reply:
Hey @mhess-swl, afaik in production this directory should always exist, so I think we're safe here.

That's an odd unit test behavior though...how did it fail?

Reply:
I figured out how to recreate it. If I run `catchesUpOnUpgradePreparationIfInContext()` by itself on the current `main` branch it fails with:
```
java.lang.IllegalArgumentException: File system element for parameter 'directory' does not exist: 'src/test/resources/upgrade'
```

Reply:
@tinker-michaelj worked out a way not to change the prod code in #4134 👍 thanks Michael!